### PR TITLE
fix of CALS-5498 RFA 1A don't store county information to database when …

### PR DESCRIPTION
**Pull request by vladimirPe**
#### Which User story does this relate to?
- https://osi-cwds.atlassian.net/browse/CALS-5498

#### Brief feature description
- Set default value for county of RFA 1a form

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
- n/a

### CODE REVIEW

**Comment by ratneshraval** [comment]
@vladimirPe Can you add test cases?

File: app/javascript/rfa_forms/rfa01a_edit_view/index.js [hunk 1]
```diff
@@ -38,6 +38,9 @@ export default class Rfa01EditView extends React.Component {
       disableSave: !(checkForNameValidation(this.props.application.applicants)),
       errors: {}
     }
+    const countyValue = getDictionaryId(this.state.application.application_county) || (this.props.user && this.props.user.county_code)
+    const county = this.props.countyTypes.find(countyType => countyType.id == countyValue)
```
Comment (ratneshraval) [inline app/javascript/rfa_forms/rfa01a_edit_view/index.js]: Line 42 need to use `===`.

File: test/javascript/rfa_forms/rfa01a_edit_view/index_test.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    derscribe('Set default value for application_county test', () => {
+        const props = {
+            user: { county_code: "51"},
+            application: {},
+            countyTypes: countyTypesArray
+        };
+        const _Rfa01EditView = shallow(<Rfa01EditView  {...props} />);
+        console.log(_Rfa01EditView.html());
+        fit('application_county', () => {
```
Comment (nasar-cambria) [inline test/javascript/rfa_forms/rfa01a_edit_view/index_test.js]: I think you forgot to remove ```f``` from the test case

File: test/javascript/rfa_forms/rfa01a_edit_view/index_test.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  const countyTypesArray = [countyType51, countyType52]
+
+  describe('Set default value for application_county test', () => {
+    const props = {
+      user: {county_code: '51'},
+      application: {},
+      countyTypes: countyTypesArray
+    }
+    const _Rfa01EditView = shallow(<Rfa01EditView {...props} />)
+    console.log(_Rfa01EditView.html())
```
Comment (ratneshraval) [inline test/javascript/rfa_forms/rfa01a_edit_view/index_test.js]: Remove the console log.

File: test/javascript/rfa_forms/rfa01a_edit_view/index_test.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import Rfa01EditView from '../../../../app/javascript/rfa_forms/rfa01a_edit_view'
```
Comment (ratneshraval) [inline test/javascript/rfa_forms/rfa01a_edit_view/index_test.js]: You don't have to traverse up, you can simply
```
import Rfa01EditView from 'rfa_forms/rfa01a_edit_view'
```

Comment (vladimirPe) [reply test/javascript/rfa_forms/rfa01a_edit_view/index_test.js]: done

File: test/javascript/rfa_forms/rfa01a_edit_view/index_test.js (hunk 3 above)
Comment (vladimirPe) [reply test/javascript/rfa_forms/rfa01a_edit_view/index_test.js]: done

**Pull request merged**